src/masonite/drivers/mail/MailSmtpDriver.py

```diff
@@ -12,29 +12,38 @@
 class MailSmtpDriver(BaseMailDriver, MailContract):
     """Mail smtp driver."""
 
+    def _message(self, message_contents=None):
+        message = MIMEMultipart('alternative')
+        message['Subject'] = self.message_subject
+        message['From'] = self.mail_from_header
+        message['To'] = self.mail_to_header
+        message['Reply-To'] = self.message_reply_to
+
+        if message_contents:
+            message_contents = MIMEText(message_contents, 'html')
+            message.attach(message_contents)
+
+        else:
+            if self.html_content:
+                message.attach(MIMEText(self.html_content, 'html'))
+
```

Comment:
The entire change here comes down to this if statement. We should keep it as `self.message_body` but do a check like:

```python
if self._mimetype == 'html':
    message.attach(MIMEText(self.message_body, 'html'))
elif self._mimetype == 'text':
    message.attach(MIMEText(self.message_body, 'plain'))

```

Reply:
Again. I want to include both content types in the same message.

Reply:
If anything I would deprecate the message_content arg. I only kept it to be backwards compatible.

Reply:
Right gotcha. You want to both both